spec/features/home_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+describe "links tests" do
+  it "opens add_a_new_card_page, after clicking on link 'Добавить карточку'" do
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.